pkg/monitor/dynkeepalived.go

```diff
@@ -98,6 +98,27 @@ func KeepalivedWatch(kubeconfigPath, clusterConfigPath, templatePath, cfgPath st
 	var appliedConfig, curConfig, prevConfig *config.Node
 	var configChangeCtr uint8 = 0
 
+	// Lease VIPS
+	// To avoid backwards compatability - Bad flow doesn't return an error
+	if clusterName, _, err := config.GetClusterNameAndDomain(kubeconfigPath, clusterConfigPath); err == nil {
+		if exists, err := needLease(cfgPath); err == nil && exists {
+			vips := []VIP{{"api", apiVip}, {"ingress", ingressVip}}
+			if err = leaseVIPs(cfgPath, clusterName, vips); err == nil {
+				log.WithFields(logrus.Fields{
+					"cfgPath":     cfgPath,
+					"clusterName": clusterName,
+					"vips":        vips,
+				}).Info("Leased VIPS successfully")
+			} else {
+				log.WithFields(logrus.Fields{
+					"cfgPath":     cfgPath,
+					"clusterName": clusterName,
+					"vips":        vips,
+				}).WithError(err).Error("Failed to lease VIPS")
+			}
+		}
+	}
+
```

Comment:
Shouldn't we monitor the lease files to see if we got addresses, and at least log if we did not?